supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/MemoryDeclarationValidation.scala

```diff
@@ -53,11 +53,12 @@ class MemoryDeclarationValidation(declaration: Declaration, attributeName: Strin
     validation.withDefault(WomLong(memorySize.bytes.toLong))
   }
 
+  case class TypeCastMismatch(message: String) extends Exception(message)
```

Comment:
Shouldn't we delare this exception class in some common module in order to keep DRY?

Reply:
I agree with @likeanowl and [his another comment](https://github.com/EpamLifeSciencesTeam/cromwell/pull/19/files#r325709371) about the same class, since the only reason this exception is used for is the validation logic of whether the value is a decimal double it would be more informative to name it accordingly such as @likeanowl suggested or even more straightforward, for instance, `NotADecimalDoubleException`.

Concerning the same class defined in the `MachineConstraints` i suggest removing it and the entire method `ceil`.